src/utils/api.js

```diff
@@ -86,13 +89,13 @@ function apiFetch(method, route, body) {
           JSON.stringify(body) :
           undefined,
     })
-    .then(attachJSON)
-    .then(response => {
-      if (response.ok) {
-        return response;
-      }
-      return Promise.reject(createAPIError(response));
-    })
+      .then(attachData)
+      .then(response => {
+        if (response.ok) {
+          return response;
+        }
+        return Promise.reject(createAPIError(response));
+      })
```

Comment:
For this and next hunk: `.then` doesn't add indentation level. In general, keep the same coding style as the rest of the project as much as possible.